src/Qdrant.Client/Grpc/VectorInput.cs

```diff
@@ -84,4 +84,34 @@ public static implicit operator VectorInput(float[][] values)
 	{
 		Id = id
 	};
+
+	/// <summary>
+	/// Implicitly converts a <see cref="Document"/> to a new instance of <see cref="VectorInput"/>
+	/// </summary>
+	/// <param name="document">an instance of <see cref="Document"/></param>
+	/// <returns>a new instance of <see cref="VectorInput"/></returns>
+	public static implicit operator VectorInput(Document document) => new()
+	{
+		Document = document
+	};
+
+	/// <summary>
+	/// Implicitly converts an <see cref="Image"/> to a new instance of <see cref="VectorInput"/>
+	/// </summary>
+	/// <param name="image">an instance of <see cref="Image"/></param>
+	/// <returns>a new instance of <see cref="VectorInput"/></returns>
+	public static implicit operator VectorInput(Image image) => new()
+	{
+		Image = image
+	};
+
+	/// <summary>
+	/// Implicitly converts an <see cref="InferenceObject"/> to a new instance of <see cref="VectorInput"/>
+	/// </summary>
+	/// <param name="obj">an instance of <see cref="InferenceObject"/></param>
+	/// <returns>a new instance of <see cref="VectorInput"/></returns>
+	public static implicit operator VectorInput(InferenceObject obj) => new()
+	{
+		Object = obj
+	};
```

Comment:
Let's comment these out for now until we release inference.

Reply:
These types would still be available though. Only these helper converters would be hidden.

Reply:
> Only these helper converters would be hidden.

That's fine for now :+1:. It makes them a bit harder to discover.

Reply:
Done.